documentation/publish/recommended-extensions.md

```diff
@@ -0,0 +1,123 @@
+
+<!-- Content with Table of Contents Module -->
+
+{% capture content_with_toc %}
+
+[Recommended extensions](https://addons.mozilla.org/firefox/search/?recommended=true&type=extension) are curated extensions that meet the highest standards of security, functionality, and user experience. Firefox staff thoroughly evaluate each extension before it receives Recommended status.
+
+Recommended extensions are easy to identify because of their distinctive “Recommended” badge.
+
+![Add-ons Manager setting cog]({% asset "documentation/publish/RecEx_badge.png" @optim @path 50%})
+
+{% endcapture %}
+{% include modules/column-w-toc.html
+  id="overview"
+  content=content_with_toc
+%}
+
+<!-- END: Content with Table of Contents -->
+
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## Criteria for Recommended extensions
+
+When evaluating nominations, curators are primarily concerned with:
+
+- **Does the extension function at an exemplary level?** All Recommended extensions should not only function as they promise, but function at a very high level. For instance, there are many ad blockers out there, but not all ad blockers are equally effective.
+
+- **Is the extension safe?** We’re committed to helping protect users against third party software that may—intentionally or otherwise—compromise user security. Before an extension receives Recommended status, it undergoes rigorous technical review by staff security experts.
+
+- **Does the extension offer an exceptional user experience?** Recommended extensions should be delightful to use. Curators look for content that’s intuitive to manage and well-designed. Common areas of concern include the post-install experience (i.e. once the user installs the extension, is it clear how to use it?), settings management, user interface copy, etc.
+
+- **Is the extension relevant to a general, international audience?** The actively managed nature of Recommended extensions means we can only feature a relatively small collection of curated extensions. We try to select Recommended extensions that will be relevant to Firefox users around the globe.
+
+The add-ons team may remove extensions from the Recommended list at any time. Reasons may include a degraded experience, security flaws, developer abandonment, or perhaps stronger alternatives emerge.
+
+{% endcapture %}
+{% include modules/one-column.html
+  id="criteria"
+  content=content
+  aside=""
+%}
+
+<!-- END: Single Column Body Module -->
+
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## Developer partnership
+
+Due to the heightened exposure Recommended extensions receive and because Firefox puts its brand endorsement behind these extensions, we only consider collaborating with developers who are committed to actively maintaining their extensions. This means:
+
+- Strict adherence to AMO’s [add-on policies](/documentation/publish/add-on-policies/). If security concerns arise, developers must be responsive addressing fixes.
```

Comment:
Should not it be 
```suggestion
- Strict adherence to AMO’s [add-on policies](/documentation/publish/add-on-policies/). If security concerns arise, developers must be responsive to addressing fixes.
```

Reply:
Not good in english, just need to make sure.

Reply:
Good catch! We'll take it.